annotation-dereferenciation/pom.xml

```diff
@@ -31,6 +31,11 @@
 			<groupId>org.springframework</groupId>
 			<artifactId>spring-core</artifactId>		
 		</dependency>
+		
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
```

Comment:
we shoulnd not make client code (metis client) dependent on web modules (spring-web). This dependency seems to have been added becasue of the spring.UriComponentBuilder. Please use the java [URIBuilder](https://docs.oracle.com/javaee/7/api/javax/ws/rs/core/UriBuilder.html) and remove this dependency